mmdet/evaluation/metrics/crowdhuman_metric.py

```diff
@@ -0,0 +1,914 @@
+            if key_tag:
+                assert key_tag in dict_input[key_name][0]
+        if key_score:
+            if key_tag:
+                bboxes = np.vstack([
+                    np.hstack((rb[key_box], rb[key_score], rb[key_tag]))
+                    for rb in dict_input[key_name]
+                ])
+            else:
+                bboxes = np.vstack([
+                    np.hstack((rb[key_box], rb[key_score]))
+                    for rb in dict_input[key_name]
+                ])
+        else:
+            if key_tag:
+                bboxes = np.vstack([
+                    np.hstack((rb[key_box], rb[key_tag]))
+                    for rb in dict_input[key_name]
+                ])
+            else:
+                bboxes = np.vstack(
+                    [rb[key_box] for rb in dict_input[key_name]])
+        bboxes[:, 2:4] += bboxes[:, :2]
+        return bboxes
+
+    def compare_voc(self, thres):
+        """
+        :meth: match the detection results with the groundtruth by VOC
+        matching strategy
+        :param thres: iou threshold
+        :type thres: float
+        :return: a list of tuples (dtbox, imageID), in the descending
+        sort of dtbox.score
+        """
+        if self.dtboxes is None:
+            return list()
+        dtboxes = self.dtboxes
+        gtboxes = self.gtboxes if self.gtboxes is not None else list()
+        dtboxes.sort(key=lambda x: x.score, reverse=True)
+        gtboxes.sort(key=lambda x: x.ign)
+
+        scorelist = list()
+        for i, dt in enumerate(dtboxes):
+            maxpos = -1
+            maxiou = thres
+
+            for j, gt in enumerate(gtboxes):
+                overlap = dt.iou(gt)
+                if overlap > maxiou:
+                    maxiou = overlap
+                    maxpos = j
+
+            if maxpos >= 0:
+                if gtboxes[maxpos].ign == 0:
+                    gtboxes[maxpos].matched = 1
+                    dtboxes[i].matched = 1
+                    scorelist.append((dt, self.ID))
+                else:
+                    dtboxes[i].matched = -1
+            else:
+                dtboxes[i].matched = 0
+                scorelist.append((dt, self.ID))
+        return scorelist
+
+
+class Database(object):
+    """Loading information and evaluation.
+
+    Args:
+        gt_path (str): Path of an annotations file.
+        dt_path (str): Path of detection results.
+        body_key (str, None): key of detection body box.
+            Valid when loading detection results and mode!=1.
+        head_key (str, None): key of detection head box.
+            Valid when loading detection results and mode!=0.
+        mode (int): Select the mode of evaluate. Valid mode include
+            0(just body box), 1(just head box) and 2(both of them).
+            Defaults to 0.
+    Examples:
+        >>> database = Database(
+        >>>     '/data/YuYoujiang/dataset/CrowdHuman/annotation_val.odgt',
```

Comment:
use a common path in examples
```suggestion
        >>>     'data/CrowdHuman/annotation_val.odgt',
```

Reply:
After merge database class into CrowdHumanMetric, it is not so convenient to calculate the metric from the saved detection results. So，this code was removed.